unified-codes/apps/web/src/components/molecules/ExplorerTablePagination.tsx

```diff
@@ -0,0 +1,29 @@
+import * as react from 'react';
+import { connect } from 'react-redux';
+
+import { TablePagination } from '@unified-codes/ui';
+
+import { TableActions, ITableAction } from '../../actions/explorer';
+import { TableSelectors } from '../../selectors/explorer';
+import { IState } from '../../types';
+
+const mapDispatchToProps = (dispatch: React.Dispatch<ITableAction>) => {
+    const onChangePage = (_: React.MouseEvent<HTMLButtonElement> | null, page: number) => dispatch(TableActions.updatePage(page));
+    const onChangeRowsPerPage = (event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>) => dispatch(TableActions.updateRowsPerPage(+event.target.value));
+    
+    return { onChangePage, onChangeRowsPerPage };
+};
+
+const mapStateToProps = (state: IState) => {
+    const rowsPerPageOptions = [25, 50, 100];
```

Comment:
OK to put some app specific logic in here, as this is an app-specific component (however, still probably better to have this in the store... I think).